jwst/outlier_detection/utils.py

```diff
@@ -78,16 +78,14 @@ def median_without_resampling(input_models,
     with input_models:
         for i in range(len(input_models)):
 
-            drizzled_model = input_models.borrow(i)
+            input_model = input_models.borrow(i)
+            drizzled_model = copy.deepcopy(input_model)
```

Comment:
```suggestion
            drizzled_model = copy.deepcopy(input_model)
```
Can we just copy the data array here? Currently the code also adds `wht` to the model but I think it's only used within this for loop so we can hopefully just save it to `wht` instead of `drizzled_model.wht`.

Reply:
Good point.  Let me try that.

Reply:
Updated to copy only the data and the WCS from the model.  

I also noticed the WCS was being copied for every input model, but only one copy is needed, so I moved it to the i=0 clause, for both resampled and not.